lib/rules/no-unknown-property.js

```diff
@@ -69,33 +69,10 @@ const ATTRIBUTE_TAGS_MAP = {
   align: ['applet', 'caption', 'col', 'colgroup', 'hr', 'iframe', 'img', 'table', 'tbody', 'td', 'tfoot', 'th', 'thead', 'tr'], // deprecated, but known
   valign: ['tr', 'td', 'th', 'thead', 'tbody', 'tfoot', 'colgroup', 'col'], // deprecated, but known
   noModule: ['script'],
-  // Media events allowed only on audio and video tags, see https://github.com/facebook/react/blob/256aefbea1449869620fb26f6ec695536ab453f5/CHANGELOG.md#notable-enhancements
-  onAbort: ['audio', 'video'],
-  onCancel: ['dialog'],
-  onCanPlay: ['audio', 'video'],
```

Comment:
eg, `onCanPlay` isn't a defensible attribute on an element that doesn't play things